libs/designsystem/modal/experimental/src/modal/modal.component.html

```diff
@@ -1,18 +1,21 @@
 <ion-modal
+  class="kirby-modal-experimental"
   [isOpen]="open"
   [canDismiss]="canDismiss"
+  [ngClass]="size"
+  [style.--height]="height"
```

Comment:
A bit cryptic - can it be change to a more comprehensible ala 'height: (user-height ? user-height : --height)' perhaps using the model?